clients/cli/src/main/java/org/apache/gravitino/cli/commands/DeleteTag.java

```diff
@@ -28,26 +29,32 @@ public class DeleteTag extends Command {
 
   protected final String metalake;
   protected final String tag;
+  protected final boolean force;
 
   /**
    * Delete a tag.
    *
    * @param url The URL of the Gravitino server.
    * @param ignoreVersions If true don't check the client/server versions match.
+   * @param force Force operation.
    * @param metalake The name of the metalake.
    * @param tag The name of the tag.
    */
-  public DeleteTag(String url, boolean ignoreVersions, String metalake, String tag) {
+  public DeleteTag(String url, boolean ignoreVersions, boolean force, String metalake, String tag) {
     super(url, ignoreVersions);
+    this.force = force;
     this.metalake = metalake;
     this.tag = tag;
   }
 
   /** Delete a catalog. */
-  @Override
   public void handle() {
```

Comment:
no need to remove the "@override" annotation I think

Reply:
Was no need - again it was a merge issue. Fixed.